examples/running_average/src/lib.rs

```diff
@@ -48,14 +49,15 @@ impl oak::Node for Node {
 }
 
 impl RunningAverageNode for Node {
-    fn submit_sample(&mut self, req: SubmitSampleRequest) {
+    fn submit_sample(&mut self, req: SubmitSampleRequest) -> GrpcResult<()> {
```

Comment:
I didn't realise Empty gets treated as (), that's neat! Was it in the protobuf code, in the gRPC code, or is it something you added?

Reply:
Partly in the [protobuf code](https://docs.rs/protobuf/2.6.2/protobuf/well_known_types/struct.Empty.html), but mostly added  in 296af915cde4f40837b1a015a6d9366b19df280a